### ISSUE TITLE
How do I shutdown a kernel when jupyter-shutdown-kernel is not found

### ISSUE THREAD

**Issue by NahsiN** [dzop/emacs-jupyter#203]
I'm running GNU Emacs 26.3 with spacemacs and the latest MELPA version of this package. How do I shutdown a kernel? The command `jupyter-shutdown-kernel` is not found when I search for it using M-x. Any ideas?

**Comment by dzop** [comment]
Yeah there isn't a default kernel shutdown command as of yet. The closest command is `C-u M-x jupyter-repl-restart-kernel`, at least inside a REPL buffer. Also, you will be asked to shutdown a kernel if you kill the REPL buffer.

Are you trying to shutdown a kernel from an Org mode buffer?
